README.md

```diff
@@ -69,3 +71,59 @@ To enable case sensitivity recursively inside the cloned directory, fire below c
 PS C:\Users\HP> cd D:\kernelci-pipeline 
 PS D:\kernelci-pipeline> (Get-ChildItem -Recurse -Directory).FullName | ForEach-Object {fsutil.exe file setCaseSensitiveInfo $_ enable}  
 ```
+
+### KCIDB setup 
+
+* add project ID to the environment file:
+```
+echo "PROJECT_ID=<your project id>" > .env
+```
+
+* add topic name to the environment file:
+```
+echo "TOPIC_NAME=<your topic name>" > .env
+```
+* add the path to your Google Application Credentials file to the environment file:
+```
+echo "GOOGLE_APPLICATION_CREDENTIALS=<your Google Application Credentials path>" > .env
+```
+
+#### Generate Google Credentials key and copy to KCIDB directory
+
+To set up and submit data to KCIDB, we need to get Google Credentials key. 
+
+[Check here to see how to get Google Credentials Key.](https://github.com/kernelci/kcidb/blob/main/doc/administrator_guide.md)
+
+NOTE: This key allows anyone to do anything with the specified Google Cloud project, so keep it safe.
+
+Once you have gotten the Google credentials key file, copy and paste it to data/kcidb 
+
+Use below command to copy the Google credentials file to data/kcidb directory.
+```
+$ cat ~/kernelci-production-admin.json  >> data/kcidb
```

Comment:
This will put the content of the JSON file in a new file called `data/kcidb`. I think you meant to use `cp` there, and then the name of the file should be set with `GOOGLE_APPLICATION_CREDENTIALS` in the env file.